mqtt/build/linux/mqtt-fuzz-rerun.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+set -euo pipefail
+
+tmux attach-session -t mqtt-fuzz && exit 0 || :
+
+DIR=$1
+if [ ! -d "$DIR" ]; then
+    echo "$DIR does not exist"
+fi
+
+cd $DIR
+
+rm -rf in/ in.cmin/ in.tmin/
+
+mkdir in.cmin/
+i=0
+for f in out/fuzzer*/queue/*; do
+    cp "$f" "in.cmin/$i"
+    i=$(( i + 1 ))
+done
+
+mkdir in.tmin/
+cargo afl cmin -i in.cmin/ -o in.tmin/ target/release/mqtt3-fuzz
+rm -rf in.cmin/
+
+mkdir in.cmin/
+for f in in.tmin/*; do
+    cargo afl tmin -i "$f" -o "in.cmin/$(basename $f)" target/release/mqtt3-fuzz
+done
+rm -rf in.tmin/
+
+mkdir in/
+cargo afl cmin -i in.cmin/ -o in/ target/release/mqtt3-fuzz
+rm -rf in.cmin/
+
+rm -rf out/
+
+(
+sleep 1
+
+tmux set-option -t 'mqtt-fuzz' pane-border-status top
```

Comment:
Sorry, remove these three `set-option` lines too. They're just personal customization (and should belong in `~/.tmux.conf` anyway).